src/styles/header.scss

```diff
@@ -4,10 +4,16 @@ header img, header nav {
 }
 
 .header {
+  position: sticky;
+  top: 0;
+  z-index: 100;
+  background-image: linear-gradient(whitesmoke 50%, rgba(245,245,245, 0));
```

Comment:
I made the background image a gradient whitesmoke and whitesmoke with 0 opacity. This creates the effect of content slowly fading under the nav bar.